### ISSUE TITLE
16.0.1 to 16.0.2 Update Failed 'update of app "accessibility"'

### ISSUE THREAD

**Issue by anyion** [nextcloud/server#16284]
Attempting to update to 16.0.2 via the web and it's failing.  "Detailed Logs" on the web update page say:

Update to 16.0.2
An error occurred.

Set log level to debug
Turned on maintenance mode
Repair step: Repair MySQL collation
Repair info: All tables already have the correct collation -> nothing to do
Repair step: Repair SQLite autoincrement
Repair step: Copy data from accounts table when migrating from ownCloud
Repair step: Drop account terms table when migrating from ownCloud
Updating database schema
Updated database
Checking for update of app "accessibility" in appstore
An error occurred.

Thoughts on how to prod the update along?  Thanks!

**Comment by anyion** [comment]
I'm was seeing this in the server logs:

{"reqId":"ydPKqJ1rscbOQslbFqIW","level":3,"time":"2019-07-07T19:43:18+00:00","remoteAddr":"70.237.XXX.XXX","user":"--","app":"PHP","method":"GET","url":"\/XXX\/core\/ajax\/update.php?requesttoken=FocJIPEd5AyNFgDTOaY46JsVgvxf7Jxd0no8l1XV%2B8Y%3D%3AdKhRSp9%2BpXm%2BIFaKbNFeuOtn0JQZnswfhz9l8D%2BAjao%3D","message":"Allowed memory size of 33554432 bytes exhausted (tried to allocate 20480 bytes) at \/home\/YYY\/public_html\/XXX\/lib\/private\/App\/AppStore\/Fetcher\/Fetcher.php#114","userAgent":"Mozilla\/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/75.0.3770.100 Safari\/537.36","version":"16.0.1.1"}

My PHP memory limit is set to 512MB but for some reason, the updater script seems to not be following my php.ini settings.

Made of copy of my php.ini file and replaced the existing one w/ the new copy, restarted the webserver and it was able to successfully update.  Strange.
